### PR TITLE
[ESD-3677] Incorrect Connection Update Example

### PR DESCRIPTION
**Pull request by ian-speers**
Fixing some minor mistakes in security/blacklisting-attributes example. This was prompted by [ESD-3677](https://auth0team.atlassian.net/browse/ESD-3677).

### CODE REVIEW

File: articles/security/blacklisting-attributes.md [hunk 1]
```diff
@@ -41,11 +41,12 @@ Here is a sample HTTP request that blacklists two attributes: `ethnicity` and `g
   "cookies": [],
   "headers": [
     { "name": "Authorization", "value": "Bearer YOUR_TOKEN" }
+    { "name": "Content-Type", "value": "application/json" }
   ],
   "queryString" : [],
   "postData": {
     "mimeType": "application/json",
-    "text": "{\"options\": {\"non_persistent_attrs\": [\"ethnicity\", \"gender\"]}\", \"enabled_clients\": [\"\"]}"
```
Comment (tamigoodall) [inline articles/security/blacklisting-attributes.md:48]: @ian-speers There seems to be an error on line 48. It's causing an error in the Heroku build. The build log indicates there is an error in the har snippet. Please double check. It looks like there may be an extra "}"

Comment (ian-speers) [reply articles/security/blacklisting-attributes.md:48]: Hey Tami, thanks for following up! Is there any additional information in the error? The previous snippet was invalid, but the changes should be correct:
```js
// old - Uncaught SyntaxError: Unexpected string in JSON at position 61
JSON.parse("{\"options\": {\"non_persistent_attrs\": [\"ethnicity\", \"gender\"]}\", \"enabled_clients\": [\"\"]}");

// new - { 'options': { 'non_persistent_attrs': ['ethnicity',  'gender'] } }
JSON.parse("{\"options\": {\"non_persistent_attrs\": [\"ethnicity\", \"gender\"]}}");
```

Comment (tamigoodall) [reply articles/security/blacklisting-attributes.md:48]: The log says: 

> Uncaught Error: Error compiling document articles/security/blacklisting-attributes.md: Invalid JSON in HAR snippet.

Comment (tamigoodall) [reply articles/security/blacklisting-attributes.md:48]: Let me try using "```js" instead of "```har" in line 36.

Comment (tamigoodall) [reply articles/security/blacklisting-attributes.md:48]: That fixed it @ian-speers. I'll merge it.

**Review by tamigoodall — APPROVED** [review]

**Pull request merged**